pkg/clients/management/management.go

```diff
@@ -0,0 +1,143 @@
+package management
```

Comment:
I specifically clarified a week or so ago that a client for the SMA would not be needed. There is nothing that calls the SMA. Instead, the SMA will be calling clients of the services it manages. This whole file should be removed. As of right now, it's only being used by tests anyway.

Reply:
Agreed. I evidently misinterpreted the pointer on the creation of "mocks" in the _client_ folder to mean that we did need the functionality after all. Again, agreed, and I will remove the file, and associated test file as well, thanks!